src/kit/ui/BooleanComponent.js

```diff
@@ -15,7 +15,7 @@ export default class BooleanComponent extends ValueComponent {
     if (!this.context.editable) {
       el.addclass('sm-readonly');
     }
-    el.append($$(CheckboxInput, { value }));
+    el.append($$(CheckboxInput, { value, disabled: this.props.disabled }));
```

Comment:
Do you need to make sure that the default for disabled is false? Is is this already handled?

Reply:
In this case everything that is falsy will work. We could cast it to boolean explicitly to enforce the type.

Reply:
I'm happy as long as it doesn't alter the default behaviour from before.

Reply:
it won't